core/src/main/java/gyro/core/scope/RootScope.java

```diff
@@ -250,34 +274,60 @@ public boolean isInWorkflow() {
 
     public Stream<String> list() {
         try {
-            return backend.list();
+            if (stateBackend != null) {
```

Comment:
Any reason why the getter is not used and the class variable directly ?

Reply:
Left as variable due to consistency with the rest of RootScope.